fixed voice numbers

Sentinel77 opened the pull request:
-Corrected voice numbers to be from 1-32 as on the DX7, instead of 0-31.

-When using -c flag, voice numbers appeared as hex. Corrected this to be in decimal.

Sentinel77 committed:
fixed voice numbers

-Corrected voice numbers to be from 1-32 as on the DX7, instead of 0-31.

-When using -c flag, voice numbers appeared as hex. Corrected this to be in decimal.

Sentinel77 commented:
fixed voice numbers

rogerallen commented:
Thanks for the pull request, but I'm not sure I agree this is fixing the issue.  Right off you can see that my simple CI checks are failing.

If the issue is that DX7 users are confused by 1-32 range vs. 0-31 one way to easily address that is via documentation that clarified this.

Internally, I think the code uses the 0-31 range in a lot more places than just on the command line.  I suspect there would be a lot of churn if we made this adjustment.

That said, anyone reading should feel free to add their opinion here.  Should we change this or just update the docs?

Sentinel77 commented:
Hi,
Thank you for your response.

Looking at the Travis checks, it appears they are simply flagging that the output has changed from what it was previously.
This is expected, as the voice numbers would now printed as 1-32 instead of 0-31.

cheers.

rogerallen commented:
Do these options still work?
```
 -b config.txt output.syx : select specific voices, breed 32 random voices
 -s config.txt output.syx : select specific voices, output to new syx file
```

Sentinel77 committed:
Update dxsyx.cpp

Ensures the -b 'breed' and -s 'new syx' functions, will still work correctly, when voice numbers are printed as 1-32.

Sentinel77 commented:
Hi,
 Thanks for pointing those out. I made a small adjustment and now those are working.
I ran through all the option combinations, and they are working correctly.

Thank you.

Sentinel77 committed:
Updated tests to use voice numbering of 1-32 (#6)

Tests updated to use voice numbering of 1-32 instead of 0-31 or hex.

Sentinel77 commented:
Hi,
 I have updated the voice numbers in the test files with values of 1-32 instead of 0-31 or hex.
All checks passed.

rogerallen commented:
Thanks, Sentinel77.  I will make time this weekend to look this over.
